src/common/health.rs

```diff
@@ -163,10 +163,10 @@ impl Task {
             // If not:
             //
             // - Check if this is the only node in the cluster
-            let Some(_) = self.cluster_commit_index().await else {
+            if self.consensus_state.peer_count() <= 1 {
```

Comment:
Is that always true when we bootstrap the peer? Assuming  that before consensus is in sync the current peer doesn't know about others?

Reply:
https://github.com/qdrant/qdrant/blob/dev/src/common/health.rs#L139-L148